src/Ledger/Prelude.agda

```diff
@@ -85,5 +85,5 @@ module Properties where
   module _ {A : Set} ⦃ _ : DecEq A ⦄ where
     open Intersectionᵖ {A} ∈-sp public
 
-_ᶠᵐ : {A B : Set} → A ↛ B → FinMap A B
+_ᶠᵐ : {A B : Set} → A ⇀ B → FinMap A B  -- unicode: `\rightharpoonup`
```

Comment:
I don't think we should leave notes on how to type something in random comments

Reply:
Okay, let's wait.